Fix/master/ro locale names

vicusbass opened the pull request:
This adds Romanian locale to Faker, person provider (gender based)

fcurella commented:
Thank you @vicusbass :)

Could you add the source of your data as comments above the list of names?

vicusbass commented:
Done, added comments.
Basically, scrapping Wikipedia and checking the list manually is one source. Myself, another source, I added some well-known Romanian last names to the list.

fcurella commented on faker/providers/person/ro_RO/__init__.py:
```diff
@@ -0,0 +1,113 @@
+        'Vasilică', 'Veniamin', 'Vicențiu', 'Victor', 'Vincențiu', 'Viorel', 'Visarion', 'Vlad', 'Vladimir', 'Vlaicu',
+        'Voicu', 'Zamfir', 'Zeno')
+
+    # sources: https://ro.wikipedia.org/wiki/Lista_celor_mai_uzuale_nume_de_familie#Rom%C3%A2nia and myself
```
Could you please remove yourself from the list? One of the reasons we want sources is to delegate the editorial responsibility away from us. By modifying the list we are making an editorial choice and we can't claim that the list comes from there anymore.

vicusbass replied:
Makes sense, removed the extra names.

fcurella commented:
Thank you! ✨
